crates/kitsune_p2p/kitsune_p2p/src/spawn/actor/space.rs

```diff
@@ -808,17 +808,17 @@ impl Space {
         let actor::RpcMulti {
             space,
             from_agent,
-            basis,
-            remote_agent_count,
-            timeout_ms,
+            //basis,
+            //remote_agent_count,
+            //timeout_ms,
             //as_race,
             //race_timeout_ms,
             payload,
             ..
         } = input;
-        let remote_agent_count = remote_agent_count.unwrap();
-        let timeout_ms = timeout_ms.unwrap();
-        let stage_1_timeout_ms = timeout_ms / 2;
+        //let remote_agent_count = remote_agent_count.unwrap();
+        //let timeout_ms = timeout_ms.unwrap();
+        //let stage_1_timeout_ms = 2000;
```

Comment:
Was this on purpose?

Reply:
there was some experimentation, i reverted it : )